cpp/include/cuspatial/experimental/ranges/multilinestring_range.cuh

```diff
@@ -153,6 +206,10 @@ class multilinestring_range {
   VecIterator _point_begin;
   VecIterator _point_end;
 
+  // TODO: find a better name
+  CUSPATIAL_HOST_DEVICE auto subtracted_part_begin();
```

Comment:
I think the semantic of this iterator is close to `segment_offset_begin()`. A prefix sum to the number of segments per linestring.

Reply:
I prefer the name `segment_offset_begin()`. I don't like "subtracted_..." for the reasons I gave in the other comment on non-standard usage.